content/en/blog/_posts/2020-12-02-dockershim-faq.md

```diff
@@ -42,7 +42,7 @@ startup if using Docker as the runtime.
 
 Given the impact of this change, we are using an extended deprecation timeline.
 It will not be removed before Kubernetes 1.22, meaning the earliest release without
-dockershim would be 1.23 in late 2021. We will be working closely with vendors
+dockershim would be 1.24 in early 2022. We will be working closely with vendors
```

Comment:
It seems to me that this logic is wrong; as I understood it, the original wording intended to make it clear that you had until the day v1.23 release as confirmed time when you could still 100% rely on having dockershim (and Docker as a container runtime) work with Kubernetes.


How about something like this instead:
```markdown
It will not be removed before Kubernetes 1.22, meaning the earliest possible release
without dockershim would be Kubenetes 1.23 in late 2021.  
_Update_: removal of dockershim is confirmed for Kubernetes v1.24, see <link to announcement>.

The Kubernetes project will be working closely with vendors
```

Reply:
Updated the PR to:
> Given the impact of this change, we are using an extended deprecation timeline.
It will not be removed before Kubernetes 1.22, meaning the earliest release without
dockershim would be 1.23 in late 2021. 
_Update_: removal of dockershim is scheduled for Kubernetes v1.24, see 
[Dockershim Removal Kubernetes Enhancement Proposal][drkep].
We will be working closely with vendors and other ecosystem groups to ensure a smooth transition and will evaluate 
things as the situation evolves.